frontend/packages/dev-console/src/components/import/GiteaIcon.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+const GiteaIcon: React.FC<React.HTMLProps<SVGElement>> = ({ style }): React.ReactElement => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 20 650 587"
+      fill="currentColor"
+      aria-labelledby="icon-title-110"
```

Comment:
```suggestion
      aria-hidden="true"
```
because now there is no id referring to `icon-title-110` in the SVG, and the icon is decorative